lib/auth/auth_with_roles.go

```diff
@@ -808,6 +809,15 @@ func (a *AuthWithRoles) NewKeepAliver(ctx context.Context) (services.KeepAliver,
 }
 
 func (a *AuthWithRoles) GenerateUserCert(key []byte, username string, ttl time.Duration, compatibility string) ([]byte, error) {
+	certs, err := a.GenerateUserCertSet(key, username, ttl, compatibility)
```

Comment:
Because `generateUserCert` below returns both the SSH and TLS certificates, you can just safely change the signature of this function to return both without adding `GenerateUserCertSet`. This method is only called by `tctl` (see RBAC check below), and `teleport` and `tctl` should be upgraded lockstep so there should not be any compatibility issues.

Reply:
great idea actually